src/styles/blocks/day.scss

```
@@ -0,0 +1,19 @@
+.day {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @include square($dayBlockSize);
+
+  background-color: $dayBlockBackground;
+  border: 1px solid $dayBorderColor;
+  font-size: 30px;
+
+  transition: background-color, transform 500ms ease-in-out;
```

Color change is not transitioned.
```suggestion
  transition: background-color 500ms ease-in-out,
    transform 500ms ease-in-out;
```

---

required fix done